.travis.yml

```diff
@@ -0,0 +1,36 @@
+language: rust
+rust:
+  - stable
+  - beta
+  - nightly-2019-05-22
+  - nightly
+
+matrix:
+  allow_failures:
+    - rust: nightly
+  fast_finish: true
+
+branches:
+  only:
+  - master
+
+cache:
```

Comment:
Are we using the default Rust caching setting from Travis or we configured sccache manually?

Reply:
We are using default rust caching by travis. We'd probably just remove or customized it, after looking at a few other open source crates:
https://github.com/seanmonstar/reqwest/blob/master/.travis.yml (no cache, there's a [PR](https://github.com/seanmonstar/reqwest/pull/259) disable cache, which actually makes build faster)
https://github.com/actix/actix/blob/master/.travis.yml (no cache)
https://github.com/rust-lang/miri/blob/master/.travis.yml (cache only `.cargo` but not `target`)